dao/src/main/resources/db/changelog/logs/ch-update-order_payment_status_translations-Hural.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="Hural-3" author="Pavlo Hural">
```

Comment:
no need to have 2 separate changesets Hural-3 and Hural-4. You can merge them into single one